encrypted-media-respec.html

```diff
@@ -2088,35 +2086,34 @@
             <li><p>Set the <var>session</var>'s <a def-id="expiration"></a> attribute to <var>expiration time</var>.</p></li>
           </ol>
         </section>
-
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
-          <p>Closing a session means that the license(s) and key(s) associated with it are no longer available to decrypt <a def-id="media-data"></a>.
-            All <a>MediaKeySession</a> methods will fail for this object.
+      
+        <section id="session-closed">
+          <h4>Session Closed</h4>
+          <p>The Session Closed algorithm updates the <a>MediaKeySession</a> state after a <a href="#key-session">session</a> has been closed.</p>
+          <p>
+            Closing a <a href="#key-session">session</a> means that the license(s) and key(s) associated with it are no longer available to
+            decrypt <a def-id="media-data"></a>. All <a>MediaKeySession</a> methods will fail for this object after this algorithm is run.
           </p>
-          <p class="note">The CDM may close a session at any point, such as in response to a <a def-id="close"></a> call, when the session is no longer needed, or when system resources are lost.
-          Keys in other sessions SHOULD be unaffected, even if they have overlapping key IDs.
+          <div class="note">
+            <p>
+              The CDM may close a session at any point, such as when the session is no longer needed or when system resources are lost.
+              In that case, this algorithm MUST be run.
+            </p>
```

Comment:
Similar to an earlier comment (https://github.com/w3c/encrypted-media/pull/184/#discussion_r62946959), it seems like it would be useful to at least note that release message(s) are not sent here (at least pending #180). Although we explicitly have a step for it, an implementation could send one using the busy-loop algorithm being discussed in #181.

Really, this could apply to any place that the session is closed, but this is the algorithm that will _always_ be called. We could word the note to cover all such cases rather than just this algorithm. Placing it after the above statement about when to run this step might also help.